x-pack/plugins/cases/public/methods/can_use_cases.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ApplicationStart } from 'kibana/public';
+import { OBSERVABILITY_OWNER, SECURITY_SOLUTION_OWNER } from '../../common/constants';
+
+export type CasesOwners = typeof SECURITY_SOLUTION_OWNER | typeof OBSERVABILITY_OWNER;
+
+export function canUseCases(capabilities: ApplicationStart['capabilities']) {
+  return (
+    owners: CasesOwners[] = [OBSERVABILITY_OWNER, SECURITY_SOLUTION_OWNER]
+  ): { crud: boolean; read: boolean } => {
+    let crud = false;
+    let read = false;
+
+    if (owners.length) {
```

Comment:
calling `some` on an empty array will always return `false` this condition is not needed

Reply:
Nice I was able to make it a lot simpler:

```
export const canUseCases =
  (capabilities: Partial<ApplicationStart['capabilities']>) =>
  (
    owners: CasesOwners[] = [OBSERVABILITY_OWNER, SECURITY_SOLUTION_OWNER]
  ): { crud: boolean; read: boolean } => {
    const crud = owners.some((owner) => capabilities[`${owner}Cases`]?.crud_cases);
    const read = owners.some((owner) => capabilities[`${owner}Cases`]?.read_cases);
    return {
      crud,
      read,
    };
  };
```
I dont think I need the nullish coalescing but would it be preferable to be explicit?